src/supporting-data/keywords/components/entry/styles/keywords-graph.module.scss

```diff
@@ -0,0 +1,32 @@
+.graph {
+  position: relative;
+  width: 500px;
```

Comment:
We tend to use the `rem` unit (except for small things) I think because it is better for respecting user zoom preferences.